bin/release-plugin-to-dot-org.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
Looks like you might have branched from another branch by accident? Might want to remove this change?

Reply:
thanks, fixed